diff --git a/node_modules/svelte-markdown/LICENSE b/node_modules/svelte-markdown/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/README.md b/node_modules/svelte-markdown/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/Parser.svelte b/node_modules/svelte-markdown/src/Parser.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/SvelteMarkdown.svelte b/node_modules/svelte-markdown/src/SvelteMarkdown.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/context.js b/node_modules/svelte-markdown/src/context.js
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/index.js b/node_modules/svelte-markdown/src/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/markdown-parser.js b/node_modules/svelte-markdown/src/markdown-parser.js
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Blockquote.svelte b/node_modules/svelte-markdown/src/renderers/Blockquote.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Br.svelte b/node_modules/svelte-markdown/src/renderers/Br.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Code.svelte b/node_modules/svelte-markdown/src/renderers/Code.svelte
old mode 100644
new mode 100755
index 2dbe1bf..f771a5a
--- a/node_modules/svelte-markdown/src/renderers/Code.svelte
+++ b/node_modules/svelte-markdown/src/renderers/Code.svelte
@@ -4,3 +4,13 @@
 </script>
 
 <pre class={lang}><code>{text}</code></pre>
+
+<style>
+  pre {
+    overflow-x: auto;
+    background-color: #1D2030;
+    color: white;
+    padding: 10px;
+    border-radius: 10px;
+  }
+</style>
\ No newline at end of file
diff --git a/node_modules/svelte-markdown/src/renderers/Codespan.svelte b/node_modules/svelte-markdown/src/renderers/Codespan.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Del.svelte b/node_modules/svelte-markdown/src/renderers/Del.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Em.svelte b/node_modules/svelte-markdown/src/renderers/Em.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Heading.svelte b/node_modules/svelte-markdown/src/renderers/Heading.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Hr.svelte b/node_modules/svelte-markdown/src/renderers/Hr.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Html.svelte b/node_modules/svelte-markdown/src/renderers/Html.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Image.svelte b/node_modules/svelte-markdown/src/renderers/Image.svelte
old mode 100644
new mode 100755
index d7ec23a..ab9d349
--- a/node_modules/svelte-markdown/src/renderers/Image.svelte
+++ b/node_modules/svelte-markdown/src/renderers/Image.svelte
@@ -5,3 +5,9 @@
 </script>
 
 <img src={href} {title} alt={text}>
+
+<style>
+  img {
+    max-width: 100%;
+  }
+</style>
diff --git a/node_modules/svelte-markdown/src/renderers/Link.svelte b/node_modules/svelte-markdown/src/renderers/Link.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/List.svelte b/node_modules/svelte-markdown/src/renderers/List.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/ListItem.svelte b/node_modules/svelte-markdown/src/renderers/ListItem.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Paragraph.svelte b/node_modules/svelte-markdown/src/renderers/Paragraph.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Strong.svelte b/node_modules/svelte-markdown/src/renderers/Strong.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Table.svelte b/node_modules/svelte-markdown/src/renderers/Table.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/TableBody.svelte b/node_modules/svelte-markdown/src/renderers/TableBody.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/TableCell.svelte b/node_modules/svelte-markdown/src/renderers/TableCell.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/TableHead.svelte b/node_modules/svelte-markdown/src/renderers/TableHead.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/TableRow.svelte b/node_modules/svelte-markdown/src/renderers/TableRow.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/Text.svelte b/node_modules/svelte-markdown/src/renderers/Text.svelte
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/src/renderers/index.js b/node_modules/svelte-markdown/src/renderers/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/svelte-markdown/types/index.d.ts b/node_modules/svelte-markdown/types/index.d.ts
old mode 100644
new mode 100755
